src/components/CreateAnOpportBody/CreateAnOpportBody.js

```diff
@@ -1,68 +1,147 @@
-import React from 'react'
+import React, { useState } from 'react'
 import "./CreateAnOpportBody.css"
 
-const CreateAnOpportBody = () => {
```

Comment:
Nice work, I was thinking of using formik, but this is should work fine as well.